queries/headers/compauth_verdicts.yml

```
@@ -0,0 +1,8 @@
+name: "Compauth verdicts"
+type: "query"
+source: |
+  map(filter(headers.hops, .authentication_results.compauth.verdict is not null),
+    .authentication_results.compauth.verdict)
```

slightly more compact, but up to you
```suggestion
  filter(map(headers.hops, .authentication_results.compauth.verdict, . is not null))
```